quote-book/src/synchronized_quote_book.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2023 MobileCoin Inc.
+
+use crate::{Error as QuoteBookError, Pair, Quote, QuoteBook, QuoteId};
+use mc_blockchain_types::BlockIndex;
+use mc_crypto_ring_signature::KeyImage;
+use mc_ledger_db::Ledger;
+use mc_transaction_extra::SignedContingentInput;
+use std::ops::RangeBounds;
+
+/// A wrapper for a quote book implementation that syncs quotes with the ledger
+#[derive(Clone)]
+pub struct SynchronizedQuoteBook<Q: QuoteBook, L: Ledger + Clone + 'static> {
+    /// List of all SCIs in the quote book, grouped by trading pair.
```

Comment:
Nit: This comment doesn't belong here - it's from an inner field of the `InMemoryQuoteBook`

Reply:
Thank you.